builds/azure-pipelines/build-release-java.yml

```diff
@@ -0,0 +1,55 @@
+trigger: none
+
+pr: none
+
+variables:
+  versionMajor: 0 # TODO When this is bumped have versionPatch counter use versionMajorMinor instead https://github.com/Azure/azure-functions-sql-extension/issues/173
```

Comment:
Charles updated the major version after bumping the sql client version in the other PR, might want to update here as well?

Reply:
We shouldn't be having duplicate stuff like that - especially for something as important as a version.

Instead you should make a top level build-release pipeline (that defines the version) with separate stages for each of the different things we build. See ADS with the mac/windows/linux pipelines we have for an example.

Reply:
Since the java library is separate from the c# extension and likely won't be needing updates very often, I plan on keeping the version separate from the nuget package. Are there concerns with this @Charles-Gagnon @MaddyDev ?

Reply:
Ok yeah that makes sense given that these are completely separate from the C# extension core. 

I even wonder if we want to auto-increment the version for this one...maybe for now we can skip doing that and just hardcode the version given that I agree it seems extremely unlikely that we'll be updating this much.